app/controllers/orders/view.js

```diff
@@ -1,9 +1,26 @@
 import Controller from '@ember/controller';
 
 export default Controller.extend({
+  isLoading: false,
+
   actions: {
     downloadInvoice() {
       window.print();
+    },
+    downloadTickets() {
+      this.set('isLoading', true);
+      this.get('loader')
+        .downloadFile(`/tickets/${this.get('model.order.identifier')}`, null, { withoutPrefix: true })
```

Comment:
Why is your URL without prefix?

Reply:
The server-side endpoint is without the prefix.

Reply:
Why?

Reply:
@CosmicCoder96 There's nothing specific actually.  I've routed the tickets to `/tickets/<order_identifier>`. Do you want me to route it to `/v1/tickets/<order_identifier>` on the server side?

Reply:
Yes, why would we change the namespace for just one endpoint?

Reply:
First change it on the server, then I will debug it.

Reply:
Yeah. I'll send a PR asap.

Reply:
@mrsaicharan1 can you post the headers of the request generated by downloadFile helper?

Reply:
`ERROR:flask_jwt:Authorization Required. Request does not contain an access token`. the headers show nothing when I print them on to the console.

Reply:
I meant on the Frontend (through chrome dev console or something similar) @mrsaicharan1

Reply:
`Host: 127.0.0.1:5000
User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:66.0) Gecko/20100101 Firefox/66.0
Accept: */*
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate
Referer: http://127.0.0.1:4200/orders/44ea48b7-d9de-4b7b-ac8f-a10fecd747d7/view
Origin: http://127.0.0.1:4200
Connection: keep-alive`

Reply:
`Host: 127.0.0.1:5000
User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:66.0) Gecko/20100101 Firefox/66.0
Accept: */*
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate
Referer: http://127.0.0.1:4200/orders/44ea48b7-d9de-4b7b-ac8f-a10fecd747d7/view
Origin: http://127.0.0.1:4200
Connection: keep-alive``Host: 127.0.0.1:5000
User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:66.0) Gecko/20100101 Firefox/66.0
Accept: */*
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate
Referer: http://127.0.0.1:4200/orders/44ea48b7-d9de-4b7b-ac8f-a10fecd747d7/view
Origin: http://127.0.0.1:4200
Connection: keep-alive`